<?xml version="1.0"?>
<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find romeo_dcm_control)/config/romeo_position_control.yaml" command="load"/>

  <!-- load the controllers -->
  <node name="romeo_controller" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" args="
     /romeo_dcm/LShoulderPitch_position_controller /romeo_dcm/LElbowYaw_position_controller /romeo_dcm/LWristYaw_position_controller /romeo_dcm/LShoulderYaw_position_controller /romeo_dcm/LElbowRoll_position_controller 
     /romeo_dcm/RShoulderPitch_position_controller /romeo_dcm/RElbowYaw_position_controller /romeo_dcm/RWristYaw_position_controller /romeo_dcm/RShoulderYaw_position_controller /romeo_dcm/RElbowRoll_position_controller 
     /romeo_dcm/LHand_position_controller /romeo_dcm/RHand_position_controller 
     /romeo_dcm/LHipYaw_position_controller /romeo_dcm/LHipRoll_position_controller /romeo_dcm/LHipPitch_position_controller /romeo_dcm/LKneePitch_position_controller /romeo_dcm/LAnklePitch_position_controller /romeo_dcm/LAnkleRoll_position_controller
     /romeo_dcm/RHipYaw_position_controller /romeo_dcm/RHipRoll_position_controller /romeo_dcm/RHipPitch_position_controller /romeo_dcm/RKneePitch_position_controller /romeo_dcm/RAnklePitch_position_controller /romeo_dcm/RAnkleRoll_position_controller
     /romeo_dcm/TrunkYaw_position_controller
     /romeo_dcm/NeckYaw_position_controller /romeo_dcm/NeckPitch_position_controller /romeo_dcm/HeadPitch_position_controller /romeo_dcm/HeadRoll_position_controller
     /romeo_dcm/joint_state_controller
     "/>
     
</launch>
